xcm/xcm-simulator/fuzzer/src/fuzz.rs

```diff
@@ -104,46 +144,71 @@ pub fn relay_ext() -> sp_io::TestExternalities {
 pub type RelayChainPalletXcm = pallet_xcm::Pallet<relay_chain::Runtime>;
 pub type ParachainPalletXcm = pallet_xcm::Pallet<parachain::Runtime>;
 
-fn run_one_input(mut data: &[u8]) {
+fn run_input(xcm_messages: [XcmMessage; 5]) {
 	MockNet::reset();
-	if let Ok(m) = Xcm::decode_all_with_depth_limit(MAX_XCM_DECODE_DEPTH, &mut data) {
-		#[cfg(not(fuzzing))]
-		{
-			println!("Executing message {:?}", m);
+
+	#[cfg(not(fuzzing))]
+	println!();
```

Comment:
What's the purpose of these empty `println`s?

Reply:
probably creating an extra newline to clean the stdout.